src/renderers/webgl/WebGLShadowMap.js

```diff
@@ -70,10 +71,12 @@ function WebGLShadowMap( _renderer, _lights, _objects, capabilities ) {
 
 		var useMorphing = ( i & _MorphingFlag ) !== 0;
 		var useSkinning = ( i & _SkinningFlag ) !== 0;
+		var useInstancing = ( i * _InstancingFlag ) !==0;
```

Comment:
Also, any thoughts on 
  
    var cluster = new THREE.InstancedMesh( geometry , material , 1000 , optionalPlacementFunction );

Should also convert regular geometry to buffer allow for that, as is it would work only with buffer.